src/modules/person/persistence/person.entity.ts

```diff
@@ -95,4 +95,8 @@ export class PersonEntity extends TimestampedEntity<PersonEntity, 'id'> {
 
     @ManyToOne({ nullable: true })
     public dataProvider?: DataProviderEntity;
+
+    @AutoMap()
+    @Property({ nullable: true })
+    public reversion?: string;
```

Comment:
Field is called 'revision' in SchulConneX not 'reversion'.
It's also required and not optional. (See personenkontext entity)